src/app/brukerprofil/BrukerprofilSide.tsx

```diff
@@ -54,26 +57,29 @@ class BrukerprofilSide extends React.Component<props> {
         if (this.props.veilederRollerReducer.status === veilederRollerReducerActionNames.INITIALIZED) {
             this.props.hentVeilederRoller();
         }
+
+        if (this.props.tilrettelagtKommunikasjonReducer.status === tilrettelagtKommunikasjonActionNames.INITIALIZED) {
```

Comment:
Kan flytte denne inn i formen for tilrettelagt kommunikasjon, kanskje?

Tanken min var at all felles data (som personobjektet og veileder-roller), kan bli hentet her, men ellers så er det kompontene selv (de ulike formene) som har ansvar for å hente sin egen data. Da får man en tydligst ansvarsfordeling hvor Brukerprofil slipper å dra inn alt selv.

Reply:
makes sense, skal ordne